README.md

```diff
@@ -53,35 +56,40 @@ Details for each force field in this repository can be found in the following ta
 
 
 ## Installation
-```bash
+
+```shell
 conda install -c conda-forge openff-forcefields
 ```
 
 ## Use
 
-Installing this package exposes an entry point that makes the `openforcefield-forcefields/offxml` directory easily accessible by other packages in the same python installation. If the [Open Force Field toolkit](https://github.com/openforcefield/openff-toolkit) is installed, it will automatically detect and use this entry point:
+Installing this package exposes an entry point that makes the `openforcefields/offxml/` directory easily accessible by other packages in the same Python installation. If the [OpenFF Toolkit](https://github.com/openforcefield/openff-toolkit) is installed, it will automatically detect and use this entry point:
```

Comment:
Good catch

Reply:
Thanks! There's a good chance it was I who introduced this error in the migration last year